src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? (
+      <p className="Person__age"> {`I am ${person.age}`} </p>
+    ) : null}
```

Comment:
```suggestion
    {person.age && (
      <p className="Person__age"> {`I am ${person.age}`} </p>
    )}
```
You don't need to use ternary operator if you return nothing in the second case